contracts/transfer/src/transfer.rs

```diff
@@ -6,16 +6,16 @@
 
 use crate::{Error, Map};
 
+use alloc::vec::Vec;
 use canonical_derive::Canon;
+use core::convert::TryFrom;
```

Comment:
This is no longer needed with rust 2021

Reply:
I didn't add, it was just "moved up" from the formatter (in fact the method doesn't use it).
Since it's not part of the scope of this PR, we should probably get rid of all of them in a separate PR.